build/WikibaseBundle/default-extra-install.sh

```diff
@@ -1,33 +1,35 @@
 #!/usr/bin/env bash
-set -ex
+set -x
```

Comment:
This one was tricky and took several hours to get to work right. The issue was the hard exit that the QuickStatements OAuth key generator would throw on re-run would cause a cascade of failure.  It might however be ok now that this is only rant on "install" now and not on every boot. Will try again...